src/main/webapp/app/components/tabs/tabs.tsx

```diff
@@ -0,0 +1,57 @@
+import React, { useEffect } from 'react';
+import { Col, Container, Row } from 'reactstrap';
+import { observer } from 'mobx-react-lite';
+import classNames from 'classnames';
+import { IRootStore } from 'app/stores/createStore';
+import { componentInject } from 'app/shared/util/typed-inject';
+import styles from './styles.module.scss';
+
+export type Tab = {
+  title: string;
+  content: React.ReactNode;
+};
+
+export interface ITabsProps extends StoreProps {
+  tabs: Tab[];
+}
+
+const Tabs = observer(({ tabs, tabsStore }: ITabsProps) => {
```

Comment:
Can we make this a function component without the store, use react hooks instead? I don't see the benefits creating the store.